card_data.py

```diff
@@ -440,29 +405,41 @@ def removeUnreleasedCards(cards):
     "Witcher": "Witcher"
 }
 
-LOCALES = ["en-US", "de-DE", "es-ES", "es-MX", "fr-FR", "it-IT", "ja-JP", "pl-PL", "pt-BR", "ru-RU", "zh-CN", "zh-TW"]
-
-TEMPLATES = getCardTemplates()
-ABILITIES = getAbilityData()
+IMAGE_URL = ""
+IMAGE_SIZES = ['original', 'high', 'medium', 'low', 'thumbnail']
+TEMPLATES = {}
+ABILITIES = {}
 TOOLTIPS = {}
-TOOLTIP_DATA = getTooltipData()
+TOOLTIP_DATA = {}
 CARD_NAMES = {}
 FLAVOR_STRINGS = {}
 
-for region in LOCALES:
-    TOOLTIPS[region] = getRawTooltips(region)
-    CARD_NAMES[region] = getCardNames(region)
-    FLAVOR_STRINGS[region] = getFlavorStrings(region)
-
-cardData = createCardJson()
-
-# Requires information about other cards, so needs to be done after we have looked at every card.
-evaluateInfoData(cardData)
-# We have to do this as well to catch cards like Botchling, that are explicitly named in the Baron's tooltip.
-evaluateTokens(cardData)
-# After the info text has been evaluated, we can extract the keywords (e.g. deploy).
-evaluateKeywords(cardData)
-removeInvalidImages(cardData)
-removeUnreleasedCards(cardData)
-# Save under v0-9-10_2017-09-05.json if the script is ran on 5 September 2017 with patch v0-9-10.
-saveJson(PATCH + "_" + datetime.utcnow().strftime("%Y-%m-%d") + ".json", cardData)
+def createCardJson(rawFolder, patch):
```

Comment:
Don't like having to pass around this rawFolder everywhere as well

Reply:
I guess it's time to define a main and drop global code execution?
```
if __name__ == "__main__":
   arguments = arguments = parse_arguments()
    main(arguments)

def main(some_args):
    do_stuff...

def parse_arguments():
    [...]
    return arguments
```
Passing around global flags is certainly an anti-pattern. Python isn't my main language so I don't know what is best pattern.